Casks/quicklook-gltf.rb

```diff
@@ -0,0 +1,12 @@
+cask 'quicklook-gltf' do
+  version :latest
+  sha256 :no_check
+
+  url 'https://github.com/toshiks/glTF-quicklook/releases/download/1.1/glTF-quicklook_v1.1.zip'
+  name 'QuickLook for gltf format files'
```

Comment:
```suggestion
  name 'glTF-quicklook'
```

This is the place for a name, not a description.